OpenEdXMobile/res/values/strings_2.xml

```diff
@@ -56,4 +56,6 @@
     <string name="label_certificate_view_button">View certificate</string>
     <!-- Title of Find Courses tab on Main Dashboard screen -->
     <string name="label_discover">Discover</string>
+    <!-- label for resume course section -->
```

Comment:
Please capitalize the first letter of `labels` in the comment and improve the overall comment in a way that someone won't confuse it with Resume/CV. 😂